README.md

```diff
@@ -329,19 +331,21 @@ jobs:
           use-ssh-source-format: true
           tag-directory-separator: /
           use-version-prefix: true
+          module-ref-mode: tag
           wiki-usage-template: |
             This is a custom wiki usage block that supports markdown.
 
             The following variables are supported:
             - {{module_name}}
             - {{latest_tag}}
+            - {{ref}}
             - {{latest_tag_version_number}}
             - {{module_source}}
             - {{module_name_terraform}}
 
             ```hcl
             module "{{module_name_terraform}}" {
-              source = "{{module_source}}?ref={{latest_tag}}"
+              source = "{{module_source}}?ref={{ref}}"
```

Comment:
add {{ref_comment}}

also let's keep latest_tag and latest_tag_version_number together, move ref after latest_tag_version_number followed by ref_comment. 

Apply same changes anywhere else (e.g. action.yml)